src/components/reporter/SubmitFindings.js

```diff
@@ -438,7 +440,18 @@ const SubmitFindings = ({ wardensList, sponsor, contest, repo }) => {
       )}
     </div>
   ) : (
-    <p>You must connect your wallet to submit findings</p>
+    <div className="login-container">
+      <div className="finding-error-container">
+        <h1>Please login</h1>
+        <p>
```

Comment:
Let's add a todo comment here to remind us that we need to include the official date once we know when this will be released

Reply:
I see your note in the PR comment, but I think it's a good idea to add a comment in the code since this PR will be closed before the full feature is released.

Reply:
Yes correct so we make sure we change it before warden auth merge, Done ✅